packages/webapp/src/_app/ui/social-button.tsx

```diff
@@ -14,6 +15,7 @@ const BASE_CLASS = "hover:underline items-center flex space-x-2";
 export const SocialButton = ({
     handle,
     color,
+    size = 6,
```

Comment:
This is sort of a hack to work around the issue where passed-in`className`s don't override those in the component. Maybe we refactor this component in the future to be more like ActionButton without too many built-in classNames. Works for now though.

Reply:
yeah, I just created #124 to tackle these dynamic namings...